raiden/network/transport/matrix/utils.py

```diff
@@ -296,6 +301,7 @@ def first_login(
         current_capabilities = ""
 
     # Only set the capabilities if necessary.
+    cap_str = capabilities_schema.load(capabilities["capabilities"])
```

Comment:
This is the piece where the tests are failing with a `KeyError`. How does it handle if capabilities is None?